app/assets/javascripts/paypal_account_settings.js

```diff
@@ -2,23 +2,28 @@ window.ST = window.ST || {};
 
 (function(module) {
 
-  module.initializePayPalAccountForm = function(formId) {
-    var form = $('#'+formId);
+
+  module.initializeNewPayPalAccountHandler = function(button_id, action) {
+    var $button = $('#'+button_id);
     var spinner = new Image();
     spinner.src = "https://s3.amazonaws.com/sharetribe/assets/ajax-loader-grey.gif";
     spinner.className = "send-button-loading-img";
 
-    form.validate({
```

Comment:
What the heck, I don't understand this... There was a form validator, but it didn't have any validations configured, only the submit handler?